src/mqt/predictor/ml/predictor.py

```diff
@@ -164,12 +177,11 @@ def generate_trainingdata_from_qasm_files(
         name_list = []
         scores_list = []
 
-        results = Parallel(n_jobs=-1, verbose=100)(
+        results = Parallel(n_jobs=1, verbose=100)(
```

Comment:
This seems like an unintentional change. I am pretty sure we want to train in parallel.

Reply:
Good catch, I reverted that change.

Reply:
you resolved this comment, but did not actually update the code...